packages/cli/src/utils/helpers.ts

```diff
@@ -53,3 +54,11 @@ export async function requireNodeIsSynced(web3: Web3) {
     failWith('Node is not currently synced. Run node:synced to check its status.')
   }
 }
+
+// SANCTIONED_ADDRESSES is so well typed that if you call includes with a string it gives a type error.
+// same if you make it a set or use indexOf so concat it with an empty string to give type without needing to ts-ignore
+const SANCTIONED_SET = new Set([''].concat(SANCTIONED_ADDRESSES))
+
+export function isSanctioned(address: string) {
+  return SANCTIONED_SET.has(address)
+}
```

Comment:
Is there a way to implement this using the [recommended API call option](https://github.com/celo-org/compliance?tab=readme-ov-file#option-1-make-an-api-call-recommended), instead of the discouraged NPM import? 

My thinking is: 

If we use the NPM import option
1. we will have to release a patch version of the CLI every time a new version of `@celo/compliance` is released.
2. users will have to update their CLI installation to the latest version to benefit from the OFAC update.

If we use the API call option: 
1. we won't have to release new versions of the CLI for every OFAC update
2. users won't have to upgrade their CLI to benefit from the OFAC update (if they are above the minimum version that has the OFAC feature)

One downside from the API call option is that the CLI has to make a network call to fetch the latest addresses from [`https://celo-org.github.io/compliance/ofac.sanctions.json`](https://celo-org.github.io/compliance/ofac.sanctions.json) every time a check is performed. Unless we include caching mechanism, which might be overkill.


What do you think?

Reply:
yeah not loving making a call every request . but also not sure how to implement caching. it seems the way to persist something when building a cli is to save a file.

Reply:
in the end i decided we could at least make it so that only 1 request per cli invocation is required by storing the result in memory.